csharp-basic-topics/ValueVsReferenceTypes/ValueVsReferenceTypes/WorkingDaysCalculator.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ValueVsReferenceTypes
+{
+    public class WorkingDaysCalculator
+    {
+        public WorkingDaysCalculator() => DaysOfTheWeek = 7;
```

Comment:
Again, why do we initialize the value of the `DaysOfTheWeek` setter in the constructor, and not on the setter directly?